lib/ama_layout/version.rb

```diff
@@ -1,3 +1,3 @@
 module AmaLayout
-  VERSION = "1.1.21"
+  VERSION = "1.3.0"
```

Comment:
Should this be going from `1.1.21` to `1.3.0` or should it be going to `1.1.22` or `1.2.0`
Should we just decide right now how this gem should be versioned?